src/rewriters/Rewriter.h

```diff
@@ -79,6 +79,12 @@ class DefaultRewriterConfig {
 public:
     virtual bool previsit(PTRef term) { return true; } // should continue visiting
     virtual PTRef rewrite(PTRef term) { return term; } // don't do anything
+    virtual bool treatIteAsVar() const { return true; }
+};
+
+class IteRewriterConfig : public DefaultRewriterConfig {
```

Comment:
I was hoping we do not need this specialization for ITEs (and remove `treatIteAsVar` from `Config`).
Can we do that or is this necessary?

Reply:
I'll remove it.